### ISSUE TITLE
Fix precompile errors on Windows

### ISSUE THREAD

**Issue by stillyslalom** [stillyslalom/PyThermo.jl#7]
```
(@v1.9) pkg> activate --temp
  Activating new project at `C:\Users\_____\AppData\Local\Temp\jl_5VWR7n`

(jl_5VWR7n) pkg> add PyThermo

[...]

Precompiling project...
  ✗ PyThermo
  6 dependencies successfully precompiled in 65 seconds. 21 already precompiled.
  1 dependency errored. To see a full report either run `import Pkg; Pkg.precompile()` or load the package
  1 dependency had warnings during precompilation:
┌ MicroMamba [0b3b1443-0f03-428d-bdfb-f27f9c1191ea]
│   Downloading artifact: micromamba
```

**Comment by stillyslalom** [comment]
Upon issuing a `precompile` command,
```
(jl_h6x2fM) pkg> precompile
Precompiling project...
  ✗ PyThermo
  0 dependencies successfully precompiled in 7 seconds. 27 already precompiled.

ERROR: The following 1 direct dependency failed to precompile:

PyThermo [6fae04a5-42e9-4d0c-90a8-26d1a9434b6b]

Failed to precompile PyThermo [6fae04a5-42e9-4d0c-90a8-26d1a9434b6b] to "C:\\Users\\_____\\.julia\\compiled\\v1.9\\PyThermo\\jl_2032.tmp".
    CondaPkg Found dependencies: C:\Users\_____\.julia\dev\PyThermo\CondaPkg.toml
    CondaPkg Found dependencies: C:\Users\_____\.julia\packages\PythonCall\1f5yE\CondaPkg.toml
    CondaPkg Found dependencies: C:\Users\_____\.julia\packages\PythonCall\1f5yE\CondaPkg.toml
    CondaPkg Found dependencies: C:\Users\_____\.julia\packages\PyThermo\SVup9\CondaPkg.toml
    CondaPkg Resolving changes
             + python
             + thermo
    CondaPkg Creating environment
             │ C:\Users\_____\.julia\artifacts\444cdd4c5cbacc31e39255b42de7eb3458f5ebd1\bin\micromamba.exe
             │ -r C:\Users\_____\.julia\scratchspaces\0b3b1443-0f03-428d-bdfb-f27f9c1191ea\root
             │ create
             │ -y
             │ -p C:\Users\_____\AppData\Local\Temp\jl_h6x2fM\.CondaPkg\env
             │ --override-channels
             │ --no-channel-priority
             │ python[version='>=3.7,<4',channel='conda-forge',build='*cpython*']
             │ thermo[version='0.1.40',channel='conda-forge']
             └ -c conda-forge
ERROR: LoadError: InitError: failed process: Process(`'C:\Users\_____\.julia\artifacts\444cdd4c5cbacc31e39255b42de7eb3458f5ebd1\bin\micromamba.exe' -r 'C:\Users\_____\.julia\scratchspaces\0b3b1443-0f03-428d-bdfb-f27f9c1191ea\root' create -y -p 'C:\Users\_____\AppData\Local\Temp\jl_h6x2fM\.CondaPkg\env' --override-channels --no-channel-priority "python[version='>=3.7,<4',channel='conda-forge',build='*cpython*']" "thermo[version='0.1.40',channel='conda-forge']" -c conda-forge`, ProcessExited(3221226505)) [3221226505]

Stacktrace:
  [1] pipeline_error
    @ .\process.jl:565 [inlined]
  [2] run(::Cmd; wait::Bool)
    @ Base .\process.jl:480
  [3] run(::Cmd)
    @ Base .\process.jl:477
  [4] _run(io::IO, cmd::Cmd, args::Any; flags::Any)
    @ CondaPkg C:\Users\_____\.julia\packages\CondaPkg\osUdN\src\resolve.jl:370
  [5] _resolve_conda_install(io::Any, conda_env::Any, specs::Any, channels::Any; create::Any)
    @ CondaPkg C:\Users\_____\.julia\packages\CondaPkg\osUdN\src\resolve.jl:271
  [6] resolve(; force::Bool, io::IO, interactive::Bool, dry_run::Bool)
    @ CondaPkg C:\Users\_____\.julia\packages\CondaPkg\osUdN\src\resolve.jl:526
  [7] resolve()
    @ CondaPkg C:\Users\_____\.julia\packages\CondaPkg\osUdN\src\resolve.jl:384
  [8] envdir()
    @ CondaPkg C:\Users\_____\.julia\packages\CondaPkg\osUdN\src\env.jl:70
  [9] init_context()
    @ PythonCall.C C:\Users\_____\.julia\packages\PythonCall\1f5yE\src\cpython\context.jl:63
 [10] __init__()
    @ PythonCall.C C:\Users\_____\.julia\packages\PythonCall\1f5yE\src\cpython\CPython.jl:21
 [11] register_restored_modules(sv::Core.SimpleVector, pkg::Base.PkgId, path::String)
    @ Base .\loading.jl:1115
 [12] _include_from_serialized(pkg::Base.PkgId, path::String, ocachepath::String, depmods::Vector{Any})
    @ Base .\loading.jl:1061
 [13] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String, build_id::UInt128)
    @ Base .\loading.jl:1506
 [14] _require(pkg::Base.PkgId, env::String)
    @ Base .\loading.jl:1783
 [15] _require_prelocked(uuidkey::Base.PkgId, env::String)
    @ Base .\loading.jl:1660
 [16] macro expansion
    @ .\loading.jl:1648 [inlined]
 [17] macro expansion
    @ .\lock.jl:267 [inlined]
 [18] require(into::Module, mod::Symbol)
    @ Base .\loading.jl:1611
 [19] include
    @ .\Base.jl:457 [inlined]
 [20] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt128}}, source::Nothing)
    @ Base .\loading.jl:2045
 [21] top-level scope
    @ stdin:3
during initialization of module C
in expression starting at C:\Users\_____\.julia\dev\PyThermo\src\PyThermo.jl:1
in expression starting at stdin:3
```

**Comment by stillyslalom** [comment]
Potentially fixed by 837b91c - will keep the issue open until fix is verified.

**Comment by stillyslalom** [comment]
Appears to be fixed, maybe with the help of d33c7e6.
